### ISSUE TITLE
blog: support lists properly in the descriptionLong frontmatter

### ISSUE THREAD

**Issue by shcheklein** [iterative/dvc.org#1224]
Right now we have to do something like:

https://github.com/iterative/dvc.org/pull/1213/files

to avoid bullet lists, since the look ugly.

**Comment by jorgeorpinel** [comment]
So support Markdown in that field?

**Comment by shcheklein** [comment]
we already support it. Just adjust CSS probably.

**Comment by jorgeorpinel** [comment]
Seems to work for me:
![image](https://user-images.githubusercontent.com/1477535/80927683-026afa00-8d65-11ea-9a2b-049d5fff2a19.png)
👇 
![image](https://user-images.githubusercontent.com/1477535/80927675-f5e6a180-8d64-11ea-9f73-04335e48e540.png)

**Comment by shcheklein** [comment]
I was not saying they were not working :) They just look ugly and not right.

**Comment by jorgeorpinel** [comment]
OK so maybe just put the rendered `descriptionLong` in some CSS class so it the regular blog content formatting (perhaps just with slightly smaller font)? Or specifically we only want bullet lists? Thanks

**Comment by shcheklein** [comment]
@jorgeorpinel not sure I understood you questions to be honest :). This ticket means adjusting CSS that we apply to this section. First step (and probably enough for this ticket) - lists (e.g. to align bullets with paragraphs, check the spacing, etc).

**Comment by julieg18** [comment]
@shcheklein, do we want the `descriptionLong` lists to look like this?  

![image](https://user-images.githubusercontent.com/43496356/136382394-9be4af9a-ab09-4651-8906-0479b1bf8482.png)

**Comment by shcheklein** [comment]
@julieg18 yep, right. I think that was the main intention here. I would check how we render lists in docs (margins, etc) though.
